chore: dfx copy updates (#586)
dfinity/docs: modules/developers-guide/pages/cli-reference/dfx-canister.adoc

> 

Reviewer: L3: replace {platform} with "Internet Computer blockchain or the local canister execution environment".

Reviewer: L29: on an {IC} network => on the Internet Computer blockchain or the local canister execution environment

Reviewer: L37: on the replica => on the Internet Computer blockchain or the local canister execution environment

> dfx canister create --all

Reviewer: L55: Overriding the default network => Overriding the default deployment option

L57: on the local network => on the local canister execution environment

----

> If you want to register unique canister identifiers for the same project on the {platform}, you can run the following command:

Reviewer: ```suggestion
If you want to register unique canister identifiers for the same project on the {platform} blockchain, you can run the following command:
```

[source,bash]
----
dfx canister --network ic create --all
----

> The {sdk-short-name} comes with an alias of `+ic+`, which is configured to point to the {platform}. You can also pass a URL as a network option, or you can configure additional aliases in `+dfx.json+` under the `+networks+` configuration.

Reviewer: ```suggestion
The {sdk-short-name} comes with an alias of `+ic+`, which is configured to point to the {platform} blockchain.
```
External users cannot use testnets, can they?

> To illustrate, you can call a canister and function running on a remote {IC} network using a command similar to the following:

Reviewer: ```suggestion
To illustrate, you can call a canister and function running on the {IC} blockchain using a command similar to the following:
```

> To illustrate, you can call a canister and function running on a remote {IC} network using a command similar to the following:

Reviewer: To be aligned with the above I'd omit this example...

[source,bash]

> ----

Reviewer: omit "the replica" in L 103

Reviewer: L209: 
Use the `+dfx canister create+` command to register one or more canister identifiers without compiled code on the {IC} network.
You must be connected to an {IC} network running locally or on a sub-network that you can access to run this command. 
=> 
Use the `+dfx canister create+` command to register one or more canister identifiers without compiled code.
You must be connected to a local canister execution environment or the Internet Computer blockchain to run this command.

== dfx canister delete

> Use the `+dfx canister delete+` command to delete a stopped canister from the local replica or on the {platform}.

Reviewer: ```suggestion
Use the `+dfx canister delete+` command to delete a stopped canister from the local canister execution environment or on the {platform} blockchain.
```

dfx canister delete hello_world
----

> To delete all of the canisters you have deployed on the `+ic+` {platform}, you can run the following command:

Reviewer: ```suggestion
To delete all of the canisters you have deployed on the {IC} blockchain, you can run the following command:
```

[source,bash]

> ----

Reviewer: L397: on the {IC} network running locally or on a sub-network that you can access.
=> on the canister execution environment running locally or on the {IC} blockchain.

Reviewer: L414: omit "the replica", optionally replace with "the canister execution environment running locally or the {IC} blockchain"

> 

Reviewer: L463: omit " as a canister on the network"

Reviewer: L495:  fix as above

== dfx canister set-controller

Use the `+dfx canister set-controller+` command to specify the identity name or principal to use as the new **controller** for a specified canister on the {platform}.
A controller identity has special rights to manage the canister it controls.
For example, only a controlling identity can be used to install, upgrade, or delete the canister under its control.

> 

Reviewer: L632: deployed on the {IC} running locally => deployed on the local canister execution environment

Reviewer: L654: local network => local canister execution environment

== dfx canister start

> Use the `+dfx canister start+` command to restart a stopped canister on the local {platform} or on a remote {platform}.

Reviewer: ```suggestion
Use the `+dfx canister start+` command to restart a stopped canister on the local canister execution environment or on the {IC} blockchain.
```

dfx canister start hello_world
----

> To start all of the canisters you have deployed on the `+ic+` {platform}, you can run the following command:

Reviewer: ```suggestion
To start all of the canisters you have deployed on the {IC} blockchain, you can run the following command:
```

== dfx canister status

> Use the `+dfx canister status+` command to check whether a canister is currently running, in the process of stopping, or currently stopped on the local {platform} or on a remote {platform}.

Reviewer: ```suggestion
Use the `+dfx canister status+` command to check whether a canister is currently running, in the process of stopping, or currently stopped on the local canister execution environment or on the {IC} blockchain.
```

dfx canister status hello_world
----

> To check the status for all of the canisters you have deployed on the `+ic+` {platform}, you can run the following command:

Reviewer: ```suggestion
To check the status for all of the canisters you have deployed on the {IC} blockchain, you can run the following command:
```

== dfx canister stop

> Use the `+dfx canister stop+` command to stop a canister that is currently running on the local {platform} or on a remote {platform}.

Reviewer: ```suggestion
Use the `+dfx canister stop+` command to stop a canister that is currently running on the local canister execution environment or on the Internet Computer blockchain.
```

dfx canister stop hello_world
----

> To stop all of the canisters you have deployed on the `+ic+` {platform}, you can run the following command:

Reviewer: ```suggestion
To stop all of the canisters you have deployed on the Internet Computer blockchain, you can run the following command:
```
